extensions/wikia/SEOTweaks/scripts/uncrawlable-urls.js

```diff
@@ -6,14 +6,19 @@ require([
 	window
 ) {
 	'use strict';
-	$(function () {
+
+	function decodeUncrawlableURL(){
 
 		// Handles middle click, ctrl+click and regular click
 		$('a[data-uncrawlable-url]').on('mousedown', function () {
 			var $this = $(this);
 			var url = window.atob($this.attr('data-uncrawlable-url'));
 			$this.attr('href', url);
 		});
-	});
+	};
+
+	$(decodeUncrawlableURL);
+
+	mw.hook('wikipage.content').add(decodeUncrawlableURL);
```

Comment:
Code looks ok. Just wondering why did you decide to use `wikipage.content` instead of `EditPageAfterRenderPreview` suggested in the ticket.

Reply:
We were talking about that with @Grunny and final decision was to use `wikipage.content` instead of `EditPageAfterRenderPreview`